cmsl1t/producers/jets.py

```diff
@@ -0,0 +1,93 @@
+from importlib import import_module
+
+DEFAULT_ATTRIBUTES = [
+    'etCorr', 'muMult', 'eta', 'phi', 'nhef', 'pef', 'mef', 'chMult',
+    'elMult', 'nhMult', 'phMult', 'chef', 'eef'
+]
+
+L1T_Jet_ATTRIBUTES = [
+    'jetEt', 'jetPhi', 'jetEta',
+]
+
+
+class Jet(object):
+    '''
+        Create a simple python wrapper for
+        L1Analysis::L1AnalysisRecoJetDataFormat
+    '''
+
+    def __init__(self, event, index, prefix='Jet_'):
+        global DEFAULT_ATTRIBUTES
+        # this could be simplified with a list of attributes
+        read_attributes = []
+        if not hasattr(event, prefix + DEFAULT_ATTRIBUTES[0]):
+            read_attributes[:] = L1T_Jet_ATTRIBUTES[:]
+            self.sortBy = 'jetEt'
+        else:
+            read_attributes[:] = DEFAULT_ATTRIBUTES[:]
+            self.sortBy = 'etCorr'
+        for attr in read_attributes:
+            setattr(self, attr, getattr(event, prefix + attr)[index])
+
+    @property
+    def sumMult(self):
+        return self.chMult + self.chMult + self.elMult
```

Comment:
This really looks like a bug, since you're really just doing `chMult *  2 + elMult`.  I think I remember this discussion somewhere and that it came from the legacy code, but do we have an issue for this and have we spoken to Shane, and the others?

Reply:
We do not have an issue for this. I expect the correct term to be `self.chMult + self.muMult + self.elMult`. However, in [JetID](https://twiki.cern.ch/twiki/bin/viewauth/CMS/JetID) neither `muMult` nor `elMult` are used. Usually you just pick charged + neutral multiplicity.